attributes/default.rb

```diff
@@ -0,0 +1,7 @@
+default['vscode']['rhel']['repository'] = 'https://packages.microsoft.com/yumrepos/vscode'
+default['vscode']['rhel']['name'] = 'code'
+default['vscode']['deb']['repository'] = 'https://packages.microsoft.com/repos/vscode stable main'
+default['vscode']['deb']['name'] = 'code'
+default['vscode']['win']['name'] = 'vscode'
+default['vscode']['osx']['name'] = 'visual-studio-code'
+default['vscode']['signing_key'] = 'https://packages.microsoft.com/keys/microsoft.asc'
```

Comment:
I'll throw an example in later on, but these don't need to be attributes, but a helper like platform_service_name in apache2, https://github.com/sous-chefs/apache2/blob/master/libraries/helpers.rb#L15

Reply:
While true in an enterprise setting where you have your own repo the signing key and repository may be different. 
(How I love/hate enterprise) but i see your point on using a case for the names of packages

Reply:
That can totally be changed via a property like an apt psckagr